### PR TITLE
Fixes cogscarab sprites not updating

### PR DESCRIPTION
**Pull request by ghost**
:cl: More Robust Than You
fix: Fixes cogscarab sprites not updating
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm [hunk 1]
```diff
@@ -237,3 +237,8 @@
 	. = ..()
 	if(.)
 		update_icons()
+
+/mob/living/simple_animal/drone/cogscarab/Life()
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: this is not an appropriate fix.

Comment (MoreRobustThanYou) [reply code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: what would be then

Comment (Xhuis) [reply code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: Maybe `handle_status_effects()`?

Comment (ChangelingRain) [reply code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm]: that's also inside of Life, though.
it should be in update_canmove.

**Pull request merged**